docs/api/api-release-notes.md

```diff
@@ -4,6 +4,11 @@ title: Release Notes
 parent: API
 nav_order: 1
 ---
+# 1.38.0
+* Add support for patient search by `birthDate` and `state-local-id`
+* Add support for duplicate detection when creating Patients
```

Comment:
Just make sure to update when you rename the params above.

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1098/commits/b335e9bae20603fd40a4b1f465c206aff9f4b9ef#diff-fbc0ac565b070ddf74f7e7e46f8c410dff362c51f8b54aba2c62aff5f74b6203R8